packages/nameguard-react/src/components/SettingsModal/SettingsModal.tsx

```diff
@@ -71,112 +71,75 @@ export const SettingsModal = () => {
                       Adjust these settings based on your preferences.
                     </legend>
                     <div className="space-y-4">
-                      <div className="relative flex items-start">
-                        <div className="flex h-6 items-center">
-                          <Checkbox
-                            id="attempt-ens-normalization"
-                            aria-describedby="attempt-ens-normalization-description"
-                            name="attempt-ens-normalization"
-                            checked={localSettings.attemptEnsNormalization}
-                            onChange={(e) =>
-                              setLocalSettings((prev) => ({
-                                ...prev,
-                                attemptEnsNormalization: e.target.checked,
-                              }))
-                            }
-                          />
-                        </div>
-                        <div className="ml-3 text-sm leading-6">
-                          <label
-                            htmlFor="attempt-ens-normalization"
-                            className="font-medium text-gray-900"
-                          >
-                            Attempt normalization
-                          </label>
-                          <p
-                            id="attempt-ens-normalization-description"
-                            className="text-gray-500 text-sm leading-5"
-                          >
-                            Attempt ENS Normalization before inspecting search
-                            queries. If normalization fails the raw search query
-                            will be inspected instead.
-                          </p>
-                        </div>
-                      </div>
+                      <Checkbox
+                        id="attempt-ens-normalization"
+                        aria-describedby="attempt-ens-normalization-description"
+                        name="attempt-ens-normalization"
+                        checked={localSettings.attemptEnsNormalization}
+                        onChange={(e) =>
+                          setLocalSettings((prev) => ({
+                            ...prev,
+                            attemptEnsNormalization: e.target.checked,
+                          }))
+                        }
+                      >
+                        <p>Attempt normalization</p>
```

Comment:
The children prop is not restricted to a single node. It can be a single node, multiple nodes, or even a string. This flexibility allows for more dynamic and composable UI structures. Having two node elements as children of the Checkbox component is perfectly fine.